docs/gettingstarted.rst

```diff
@@ -69,13 +37,18 @@ Are you looking at installing and deploying REANA cluster locally on your laptop
       $ # install reana-cluster utility
       $ pip install reana-cluster
 
-5. Start REANA cluster instance on Minikube:
+4. Start REANA cluster instance on Minikube:
 
    .. code-block:: console
 
       $ reana-cluster init
 
-6. Check the status of the REANA cluster deployment. (Note that it may take
+    .. note::
```

Comment:
What do you think about changing the order here? `reana-cluster init --traefik` in the console block and then mention in the note block that REANA needs an [Ingress Controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) for the interactive sessions feature to work, that a custom one can be set up but we recommend traefik configured with the `--traefik` flag.

I think of this mainly because if you ignore the note then you will get a cluster in which interactive sessions do not work.